handlers.go

```diff
@@ -107,6 +108,24 @@ var StartVMMHandler = Handler{
 	},
 }
 
+// CreateLogFilesHandler is a named handler that will create the fifo log files
+var CreateLogFilesHandler = Handler{
+	Name: CreateLogFilesHandlerName,
+	Fn: func(ctx context.Context, m *Machine) error {
+		logFifoPath := m.cfg.LogFifo
+		metricsFifoPath := m.cfg.MetricsFifo
+
+		if err := createFifos(logFifoPath, metricsFifoPath); err != nil {
```

Comment:
We should check whether or not any of the paths are empty. If they are logging is disabled.